manifests/t/TechSmith/Camtasia/23.2.0.47710/TechSmith.Camtasia.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TechSmith.Camtasia
+PackageVersion: 23.2.0.47710

```

```suggestion
PackageVersion: 23.2.0.47710
MinimumOSVersion: 10.0.19042.0
```

Install failed on my machine due to OS version
> [14BC:14C0][2023-08-23T09:34:32]i000: #@# Launch Condition not met, Condition: 'NOT Version9X AND VersionNT >v6.2 AND (WINDOWS_BUILD_NUMBER >= MinimumWindows10BuildNumber)', Message: 'Camtasia 2023 requires Microsoft Windows 10 version 20H2 or newer to install.  Please update Windows based on our <a href="https://www.techsmith.com/redirect.asp?target=systemrequirements&product=camtasiastudio&ver=23.2.0&lang=ENU&os=win">system requirements</a> before trying your installation again.' #@#